src/components/ColorPicker/components/ColorPickerContent/ColorPickerContent.tsx

```diff
@@ -1,40 +1,69 @@
 import cx from "classnames";
 import { difference as _difference, intersection as _intersection } from "lodash-es";
-import PropTypes from "prop-types";
 import React, { forwardRef, useCallback, useMemo, useRef } from "react";
-import { SIZES } from "../../../../constants/sizes";
-import { COLOR_STYLES, contentColors } from "../../../../utils/colors-vars-map";
+import { BASE_SIZES, BASE_SIZES_VALUES } from "../../../../constants";
+import { COLOR_STYLES, CONTENT_COLORS_VALUES, contentColors } from "../../../../utils/colors-vars-map";
 import NoColor from "../../../Icon/Icons/components/NoColor";
-import { COLOR_SHAPES, DEFAULT_NUMBER_OF_COLORS_IN_LINE } from "../../ColorPickerConstants";
+import { COLOR_SHAPES, COLOR_SHAPES_VALUES, DEFAULT_NUMBER_OF_COLORS_IN_LINE } from "../../ColorPickerConstants";
 import { calculateColorPickerWidth } from "../../services/ColorPickerStyleService";
-import "./ColorPickerContentComponent.scss";
+import "./ColorPickerContent.scss";
 import {
   GridKeyboardNavigationContext,
   useGridKeyboardNavigationContext
 } from "../../../GridKeyboardNavigationContext/GridKeyboardNavigationContext";
 import { ColorPickerClearButton } from "./ColorPickerClearButton";
 import { ColorPickerColorsGrid } from "./ColorPickerColorsGrid";
+import { VibeComponentProps, VibeComponent } from "../../../../types";
 
-const ColorPickerContentComponent = forwardRef(
+export interface ColorPickerContentProps extends VibeComponentProps {
+  value: string | string[] | CONTENT_COLORS_VALUES | CONTENT_COLORS_VALUES[]; //TODO - make sure this is correct
+  onValueChange: (value: CONTENT_COLORS_VALUES[] | string[]) => any; //TODO - make sure this is correct
+  colorsList: CONTENT_COLORS_VALUES[] | string[];
+  ColorIndicatorIcon?: ({ size, className }: { size?: string; className?: string }) => JSX.Element;
+  SelectedIndicatorIcon?: ({ size, className }: { size?: string; className?: string }) => JSX.Element;
+  NoColorIcon?: ({ size, className }: { size?: string; className?: string }) => JSX.Element;
+  colorStyle?: "regular" | "selected";
```

Comment:
I think it could look better if we used enums here for colorStyle, colorSize, colorShape. WDYT?

Reply:
The `COLOR_STYLES` looks like this:
```tsx
export const COLOR_STYLES = {
  REGULAR: "regular",
  HOVER: "hover",
  SELECTED: "selected"
} as const;
```
It contains also the "hover" state which isn't a valid style for the component.
There are a few options:
1. create a common type only for the `ColorPicker` component
2. create a common type in the general types file, but this makes sense only if there are more such cases.
3. keep it as-is since the inner function that uses it `getMondayColorAsStyle` expect to get one for the `COLOR_STYLES` keys, so it is type-safe like this.

I think that option 1 is good enough for now, but let me know if you think it should be something else